scripts/library.sh

```diff
@@ -448,8 +442,34 @@ function get_canonical_path() {
   echo "$(cd ${path%/*} && echo $PWD/${path##*/})"
 }
 
+# Return the version for the current branch.
+function get_branch_version() {
+  local branch_name="$(git rev-parse --abbrev-ref HEAD)"
+  # Return the actual version number for the release branch.
+  # For example, we will return "0.5" if the branch name is "release-0.5".
+  if [[ $name =~ ^release-[0-9\.]+ ]]; then
+    echo ${branch_name} | sed 's/release-//g'
```

Comment:
I think it's safer if, once in a branch, you find the latest tag and use it; e.g. in branch release-0.5 you would use the tag v0.5.2.

Reply:
Sorry I didn't quite get your point. Shouldn't  the yaml files we use here correspond to the code we use to run test? Or do you mean when `Prow` runs the CI jobs for branch release-0.5, it actually uses the code tagged v0.5.2?

Reply:
Yes, they should. Prow runs on the latest commit of a branch, but releases are tagged, so the best approximation is using the latest tag.

Reply:
Thanks for the explanation. Done.